src/codegen/ruby/def/field.rb

```diff
@@ -21,7 +21,7 @@ def java_name
     
     prefix = "is" if type.start_with?('bool')
   
-    prefix + name.tr('?$', '').split('_').collect(&:capitalize).join
+    prefix + name.tr('-?$', '_').split('_').collect(&:capitalize).join
```

Comment:
What's the relevance of this? What does it do?

Reply:
I had to add this because the name `delete-message-days`

It replaces - with _, removes ? and &, and then changes it from snake_case to camelCase

Reply:
Gotcha, don't know much about regex with ruby.

Reply:
This one's not actually regex. It's the same as `tr`: http://linuxcommand.org/lc3_man_pages/tr1.html

Reply:
Ah alright, that's interesting, I've never seen that. I'll definitely look more into that.